src/bessel.jl

```diff
@@ -613,3 +614,37 @@ hankelh1x(nu, z) = besselhx(nu, 1, z)
 Scaled Bessel function of the third kind of order `nu`, ``H^{(2)}_\\nu(x) e^{x i}``.
 """
 hankelh2x(nu, z) = besselhx(nu, 2, z)
+
+function _besselhs(nu::Float64, k::Int32, z::Complex{Float64}, N::Int32)
+    ai1, ai2 = Array{Float64}(N), Array{Float64}(N)
```

Comment:
For 0.7 compatibility, the array constructors should instead be `Vector{Float64}(undef, N)`.

Reply:
Done.